dist/filter-menu/ds4/filter-menu.css

```diff
@@ -0,0 +1,154 @@
+span.filter-menu-aria,
+span.filter-menu-form {
```

Comment:
At first glance I wasn't keen on these names, thinking them to be too implementation specific. I was expecting them to be `filter-menu` and `fake-filter-menu`. I'm starting to like these names though, even if they do break with tradition a bit, I wonder what others think...

Reply:
How about `filter-menu` and `filter-form`? @agliga @msendlakowski

Reply:
`filter-menu` and `filter-menu--form`?

Reply:
I definitely don't like "aria".  It doesn't tell me what it's used for.

I'm not sure using a BEM modifier is the right way though.  I think of a modifier as a variant on the original style.  But this is really a whole new implementation, with the same style.  Also, children would get complex, since items would have to be `filter-menu--form__item`.

I like `filter-menu` and `filter-menu-form`?  You know they are related, but they are their own beasts.

Reply:
I've gone with `filter-menu` and `filter-menu-form`. We can update if needed in another PR.

Reply:
I've gone with `filter-menu` and `filter-menu-form`. We can update if needed in another PR.